packages/database/ios/RNFBDatabase/RNFBDatabaseQuery.m

```diff
@@ -130,7 +130,9 @@ - (void)removeEventListener:(NSString *)eventRegistrationKey {
 }
 
 - (void)removeAllEventListeners {
-  for (NSString * eventRegistrationKey in _listeners) {
+  NSDictionary *listeners = [_listeners copy];
```

Comment:
Better to use `allKeys` instead of copying the entire dictionary. Can we close this PR in favour of https://github.com/invertase/react-native-firebase/pull/2772 as this uses `allKeys`?

Reply:
I see that CLA is not agreed on in that PR, so thought I just update this PR. Is that ok?

Reply:
Sure go for it

Reply:
Great. Waiting for your approval then.